src/igvfd/schemas/analysis_set.json

```diff
@@ -56,8 +56,15 @@
         }
     },
     "fuzzy_searchable_fields": [
+        "@type",
+        "assay_title",
         "dbxrefs",
-        "@type"
+        "donors",
+        "files",
+        "input_file_sets",
+        "lab.title",
+        "samples",
```

Comment:
for samples and donors, I was thinking that we would want donors.accession donors.id, samples.accession, samples.term_name